ignore/src/overrides.rs

```diff
@@ -140,7 +140,8 @@ impl OverrideBuilder {
 
     /// Toggle whether the globs should be matched case insensitively or not.
     /// 
-    /// This is disabled by default.
+    /// This is disabled by default. In order for case sensitivity to work, this function
+    /// must be called before any overrides are added.
```

Comment:
I think we can be more precise here:

```
/// When this option is changed, only globs added after the change will be affected.
///
/// This is disabled by default.
```

Also, can you add this to the gitignore builder too?